src/main/java/org/mule/extension/api/introspection/ImmutableOperationModel.java

```diff
@@ -38,12 +49,39 @@ public ImmutableOperationModel(String name,
                                    List<ParameterModel> parameterModels,
                                    MetadataType returnType,
                                    MetadataType attributesType,
-                                   Set<ModelProperty> modelProperties)
+                                   Set<ModelProperty> modelProperties,
+                                   MetadataResolverFactory metaDataResolverFactory)
     {
         super(name, description, modelProperties, parameterModels);
+        this.metadataResolverFactory = metaDataResolverFactory;
 
         this.returnType = returnType;
         this.attributesType = attributesType;
+        this.parameterModels = Collections.unmodifiableList(parameterModels);
+
+        this.content = parameterModels.stream()
+                .filter(p -> p.getModelProperty(MetadataModelProperty.class).isPresent() &&
+                             p.getModelProperty(MetadataModelProperty.class).get().isContent())
+                .findFirst();
+    }
+
+    private static boolean overridesMethod(Class<?> childClass, Class<?> parentClass, String methodName)
```

Comment:
doesn't belong here... At the very least should be in IntrospectionUtils... What's this for in the first place?